### DISCUSSION TITLE
Error: x-deno-subhost header must be set

### DISCUSSION THREAD

**Discussion by NicoCaldo** [denoland/deno#26658] [Q&A]
After correctly deploying an edge function using Deno, I'm seeing an error when it's called `Error: x-deno-subhost header must be set `

What is it? I can't find anything that cites this error/header

**Comment by igorbdl** [comment]
Hey @NicoCaldo, could you please clarify in which provider this edge function is deployed? Many thanks

**Comment by NicoCaldo** [comment]
The function is deployed on Supabase Edge Function and it looks like

```
import { Hono } from "jsr:@hono/hono";
import { serve } from "https://esm.sh/inngest@3.22.13/hono";
import { functions, inngest } from "./inngest.ts";

const functionName = '<function-name>'
const app = new Hono().basePath("/${functionName}")

app.on(
  ["GET", "PUT", "POST"],
  "/api/inngest",
  serve({
    client: inngest,
    functions,
  }),
);

Deno.serve(app.fetch);
```

it manages an [Inngest ](https://www.inngest.com/) deployment. When I call an Inngest function the error occurred. I'm trying to understand where the error comes from. If it's the Inngest backend that can't connect to the Edge Function for some reason or the other way around
